lib/pbench/test/unit/server/test_cache_manager.py

```diff
@@ -153,9 +153,10 @@ def fake_metadata_controller(tar_path):
         # fetching metadata from metadata.log file and key/value not
         # being there should result in a MetadataError
         source_tarball, source_md5, md5 = tarball
+        dataset_name = Dataset.stem(source_tarball)
         cm = CacheManager(server_config, make_logger)
 
-        expected_metaerror = f"A problem occurred processing metadata.log from {source_tarball!s}: 'Invalid Tarfile'"
+        expected_metaerror = f"A problem occurred processing metadata.log from {source_tarball!s}: \"A problem occurred processing '{dataset_name}/metadata.log' from {source_tarball!s}: Invalid Tarfile\""
```

Comment:
Does the message really end up with almost identical text appearing twice?  That seems wrong....